src/games.py

```diff
@@ -0,0 +1,140 @@
+import db, utils, config
```

Comment:
Unneeded config import

Reply:
This is actually used (e.g. on line 135), but it's confusing to both `import config` (and use config.GAME_ANNOUNCE_TIME) and also `from config import ADMIN_ACCESS` (and use ADMIN_ACCESS directly).

Good catch - I'll standardize the import format. Do you prefer the first or second one?

Reply:
I've been using the second if it's only a couple of imports

Reply:
👍